chapters/convertToAbbreviatedForm.typ

```diff
@@ -135,49 +101,156 @@ They will be stored as a member of the tweak object and then used during the app
     ],
   ),
   caption: "Elements captured for the second refactoring",
-)
+) <second_refactoring_captured_elements_figure>
 
-#pagebreak()
-=== Prerequisites
+=== Call Site Implications <call_site_implications>
+The refactoring must not change the signature of the target method.
+In regard to this specific refactoring, the order of type parameters must stay the same.
+This is only the case if the auto parameters are in the same order as their original template parameters.
+
+For example the two methods in @call_site_differences result in two different signatures.
+When calling these methods with ```cpp foo<int, float>(1.0, 2)``` only the version on the left would compile,
+as the parameter types would be `int` for `param1` and `float` for `param2`.
+The second version would be the opposite, `int` for `param1` and `float` for `param2`, which then breaks the call as the parameter types don't match.
```

Comment:
@jeremystucki irgendwo hesch 'does not' drin, mer müsstet eus einige eb mer wennd abschürze oder ned

Reply:
Evtl. chent mers na besser ufzeige ih dem mer für rechts und lings na d params ihsetzt. Oder irgendwie visualisiert, ned zwingend, mer würs aber glaub besser gseh

```cpp
// foo<int, float>(1.0, 2) 
// auto foo(float param1, int param2)
```

Reply:
im moment chömmers susch so lah aber finds chli schwierig z verstah wenn im text "the version on the left" anstatt das irgendwie z visualiiere. Isch eifach en Vorschlag, chans ja evtl. spöter ahluege und en konkrete Vorschlag mache :)